scripts/core/ui/components/ActionButton.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+
+interface IProps {
+    ariaValue: string;
+    refValue?(event): void;
```

Comment:
it's not a value if it's a callback, but calling it just `ref` would be probably ok